<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>primaryMathTestClass</class>
 <widget class="QMainWindow" name="primaryMathTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>384</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>433</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>primaryMathTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="1">
     <widget class="QPushButton" name="scorePushButton">
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit234_26"/>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="lineEdit663_13"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label375_25">
        <property name="text">
         <string>5 .375 ÷ 25 =</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEdit481_13">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit125_5"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label663_13">
        <property name="text">
         <string>3. 663 ÷ 13 =</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label525_21">
        <property name="text">
         <string>8. 525 ÷ 21 =</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit375_25">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label125_5">
        <property name="text">
         <string>1. 125 ÷ 5 =</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label986_34">
        <property name="text">
         <string>7. 986 ÷ 34 =</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit525_21">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="lineEdit897_23">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label702_9">
        <property name="text">
         <string>2. 702 ÷ 9 =</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit702_9"/>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label481_13">
        <property name="text">
         <string>6. 481 ÷ 13 =</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label234_26">
        <property name="text">
         <string>4. 234 ÷ 26 =</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit986_34"/>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label897_23">
        <property name="text">
         <string>9. 897 ÷ 23 =</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label726_33">
        <property name="text">
         <string>10. 726 ÷ 33 =</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit726_33"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="wrongLabel">
        <property name="text">
         <string>错题</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="scoreLable">
        <property name="text">
         <string>得分</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="wrongLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="scoreLineEdit"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>输入口令获取答案：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="answerLabel">
        <property name="text">
         <string>答案：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inputLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="answerPushButton">
        <property name="text">
         <string>提交</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QTextEdit" name="answerTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>433</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="primaryMathTest.qrc"/>
 </resources>
 <connections/>
</ui>
